src/main/kotlin/org/jmailen/gradle/kotlinter/tasks/FormatTask.kt

```diff
@@ -39,7 +39,7 @@ open class FormatTask @Inject constructor(
 
         workQueue.submit(FormatWorkerAction::class.java) { p ->
             p.name.set(name)
-            p.files.from(source)
+            p.files.from(getChangedSources(inputChanges))
```

Comment:
I'm not certain we can apply the same incremental logic to formatKotlin as it changes files itself 🤔. I remember running into problems with this at some point when I tried to make format more incremental (at the sourceSet level) than it was.

Reply:
`getChangedSources` returns changed files only when the task is invoked incrementally, which is never the case for `formatKotlin` so the `getChangedSources` will always return whole set of source files. 
But I see how that might be confusing, I'll consider keeping old behavior to be more explicit on the difference between these two tasks types 👍